src/cards/ha-weather-card.js

```diff
@@ -152,8 +185,10 @@ class HaWeatherCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
               >
                 <div>
                   [[localize('ui.card.weather.attributes.wind_speed')]]:
-                  [[getWind(stateObj.attributes.wind_speed,
-                  stateObj.attributes.wind_bearing, localize)]]
+                  <div class="unit">
```

Comment:
Windspeed is not a unit?

Reply:
It is a number with a unit... that's the idea

Reply:
Then pick a more descriptive name

Reply:
Changed to `measurand`